### PR TITLE
Cleaning up ZK etcd related things

### PR DESCRIPTION
**Pull request by chr0n1x**
:(

### CODE REVIEW

File: v3/fleet-local/control/chronos@.service [hunk 1]
```diff
@@ -6,12 +6,12 @@ Requires=docker.service
...
+Environment=ZOOKEEPER_HOST=localhost:2181
 Environment="IMAGE=etcdctl get /images/chronos"
-Environment="zk_username=etcdctl get /zookeeper/ZK_USERNAME"
-Environment="zk_password=etcdctl get /zookeeper/ZK_PASSWORD"
-Environment="principalframework=etcdctl get /frameworkauth/username"
-Environment="zookeeper_endpoint=etcdctl get /environment/ZOOKEEPER_ENDPOINT"
+Environment="ZK_USERNAME=etcdctl get /zookeeper/username"
+Environment="ZK_PASSWORD=etcdctl get /zookeeper/password"
+Environment="ZK_ENDPOINT=etcdctl get /zookeeper/endpoint"
+Environment="MESOS_USERNAME=etcdctl get /frameworkauth/username"
```
Comment (matthewdfuller) [inline v3/fleet-local/control/chronos@.service:14]: So, should we put these in `zookeeper/config/values`? To keep with our `config` format

Comment (chr0n1x) [inline v3/fleet-local/control/chronos@.service:14]: OH DERP

Comment (chr0n1x) [inline v3/fleet-local/control/chronos@.service:14]: the fail is so real

**Comment by matthewdfuller** [comment]
👍

**Comment by matthewdfuller** [comment]
When merging, this PR must be merged at the same time: https://github.com/adobe-platform/infrastructure/pull/69

**Comment by bossjones** [comment]
👍

**Pull request merged**